pkg/k8s/test/integration/cluster_summary_test.go

```diff
@@ -0,0 +1,60 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/aquasecurity/trivy/pkg/commands"
+	"github.com/aquasecurity/trivy/pkg/flag"
+	"github.com/aquasecurity/trivy/pkg/k8s/report"
+	"github.com/stretchr/testify/assert"
+)
+
+// Note: the test required k8s (kind) cluster installed
+
+func Test_ExecuteK8sClusterScanVulns(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	actual := new(bytes.Buffer)
+	commands.SetOut(actual)
+	globalFlags := flag.NewGlobalFlagGroup()
+	rootCmd := commands.NewRootCommand("k8s-test", globalFlags)
+	k8s := commands.NewKubernetesCommand(globalFlags)
+	rootCmd.AddCommand(k8s)
```

Comment:
You can use this function.
https://github.com/aquasecurity/trivy/blob/48e021ea6bf21125d92cb3562b74db121972e17a/integration/integration_test.go#L188-L196

Reply:
You need to add `k8s_integraion` here.
https://github.com/aquasecurity/trivy/blob/48e021ea6bf21125d92cb3562b74db121972e17a/integration/integration_test.go#L1